diff -up lxqt-admin-2.2.0/CMakeLists.txt.omv~ lxqt-admin-2.2.0/CMakeLists.txt
--- lxqt-admin-2.2.0/CMakeLists.txt.omv~	2025-10-12 14:01:47.629418864 +0200
+++ lxqt-admin-2.2.0/CMakeLists.txt	2025-10-12 14:02:10.970318725 +0200
@@ -15,6 +15,7 @@ set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}" C
 set(LXQT_MINIMUM_VERSION "2.2.0")
 set(QT_MINIMUM_VERSION "6.6.0")
 
+find_package(Qt6 ${QT_MINIMUM_VERSION} REQUIRED COMPONENTS GuiPrivate)
 find_package(Qt6Widgets ${QT_MINIMUM_VERSION} REQUIRED)
 find_package(lxqt ${LXQT_MINIMUM_VERSION} REQUIRED)
 find_package(PolkitQt6-1 REQUIRED)
